doc/classes/TabBar.xml

```diff
@@ -323,6 +323,9 @@
 		</constant>
 	</constants>
 	<theme_items>
+		<theme_item name="drop_mark_color" data_type="color" type="Color" default="Color(1, 1, 1, 1)">
+			[theme_item drop_mark] icon modulate color.
```

Comment:
Starting a sentence with a reference, which is lower case, might look a bit weird.
```suggestion
			Modulation color for the [theme_item drop_mark] icon.
```